lms/templates/ux/reference/instructor_dashboard/cohorts.html

```diff
@@ -0,0 +1,188 @@
+                <h2 class="group-header-title"><span class="title-value">Cohort Name Can be Placed Here and Should Accommodate Almost Everything</span> <span class="group-count">(contains 12,546 students)</span></h2>
+                <div class="cohort-management-group-setup">
+                  <p class="cohort-management-group-text">Students are added to this group only when you provide their email addresses or usernames on this page.<br /><a href="#">What does this mean?</a></p>
+                  <div class="cohort-management-group-actions">
+                    <span class="float-right"><a href="#">Edit settings in Studio</a></a>
+                  </div>
+                </div>
+                <ul class="wrapper-tabs">
+                  <li class="tab"><a href="#">Manage Students in Cohort Groups</a></li>
+                  <li class="tab is-selected"><a href="#">Cohort Group Settings</a></li>
+                </ul>
+
+                <div class="cohort-management-details">
+                  <form action="" method="post" name="" id="cohort-management-details-form" class="cohort-management-details-form">
+                    <h3 class="form-title sr">Cohort Group Details</h3>
+                    <div class="form-fields">
+                      <div class="cohort-management-details-name field field-text">
+                        <h4 class="cohort-section-header no-top-margin">Cohort Group Name:</h4>
+                        <label for="cohort-details-name" class="label sr">Cohort Group Name</label>
+                        <input type="text" name="cohort-details-name" value="Cohort Name is Placed Here and Should Accommodate Almost Everything" class="input cohort-details-name" id="cohort-details-name" placeholder="Enter Your Cohort's Name" required />
+                      </div>
+                    </div>
+
+                    <h4 class="cohort-section-header divider-lv2">Students in this cohort group are:</h4>
+                    <div class="form-fields">
+                      <div class="cohort-management-details-assignment field field-radio">
+                        <label for="cohort-assign-random"><input type="radio" name="cohort-assignment" value="random" /> Randomly Assigned</label>
+                        <label for="cohort-assign-manual"><input type="radio" name="cohort-assignment" value="menual" checked="checked" /> Manually Assigned</label>
+                      </div>
+                    </div>
+
+                    <!-- When options are available but a deleted one is selected. -->
+                    <h4 class="cohort-section-header divider-lv2">Is this cohort group associated with course-based content groups?</h4>
+                    <div class="form-fields">
+                      <div class="cohort-management-details-assn-course field field-radio">
+                        <label for="cohort-assn-course-no"><input type="radio" name="cohort-assn-course" value="no" /> No</label>
```

Comment:
no need for ```for``` attribute.  It's implied and currently non-conformant (takes value of ID, not class)